packages/esm-service-queues-app/src/queue-table/cells/queue-table-priority-cell.component.tsx

```diff
@@ -1,16 +1,11 @@
 import React from 'react';
 import QueuePriority from '../../queue-entry-table-components/queue-priority.component';
 import { type QueueTableColumnFunction, type QueueTableCellComponentProps } from '../../types';
-import { type PriorityColumnConfig } from '../../config-schema';
 
-export const queueTablePriorityColumn: QueueTableColumnFunction = (key, header, config: PriorityColumnConfig) => {
+export const queueTablePriorityColumn: QueueTableColumnFunction = (key, header) => {
   const QueueTablePriorityCell = ({ queueEntry }: QueueTableCellComponentProps) => {
     return (
-      <QueuePriority
-        priority={queueEntry.priority}
-        priorityComment={queueEntry.priorityComment}
-        priorityConfigs={config?.priorityConfigs}
-      />
+      <QueuePriority priority={queueEntry.priority} priorityComment={queueEntry.priorityComment} priorityConfigs={[]} />
```

Comment:
It looks like we're no longer passing `priorityConfigs` from the config schema down to `QueuePriority`. That seems wrong. Shouldn't this be:

```tsx
const QueueTablePriorityCell = ({ queueEntry }: QueueTableCellComponentProps) => {
    const config = useConfig<ConfigObject>();
    return (
      <QueuePriority
        priority={queueEntry.priority}
        priorityComment={queueEntry.priorityComment}
        priorityConfigs={config.priorityConfigs}
      />
    );
  };
```